packages/itwinui-css/src/backdrop/backdrop.scss

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-backdrop {
```

Comment:
Would be nice to also have progress backdrop here too but maybe in the future.

Reply:
I think "backdrop" is kinda different from "overlay" which is what the progress one is. Backdrop goes _under_ something else (e.g. dialog or hamburger menu).